### ISSUE TITLE
Meaning of output pose vector in humanML3D representation

### ISSUE THREAD

**Issue by SreehariRajan** [RobinWitch/SynTalker#6]
I'm confused about each dimension of the output 623-dimensional pose vector in humanML3D representation, as humanML3D representation has a dimension of 263. Can you provide some information regarding that? Also, how can this pose vector be converted to axis angle ?

**Comment by RobinWitch** [comment]
1.The reason the feature dimension is 623 is that we are using the SMPL-X format instead of SMPL-H. Exactly, we have an additional 30 joints for the hands, which is why our data representation has 623 dimensions. And we still use the feature extraction code from HumanML3D to obtain the data.

2.You can refer to our [readme.md](https://github.com/RobinWitch/SynTalker/blob/c759d0136129dac23b3f7b29ab02693499253b12/README.md?plain=1#L129). Like most existing text-to-motion works based on HumanML3D representation, we use the method from the "human body prior" repository. The pose vector includes joint spatial positions. Through optimization, we can transform the joint spatial positions into joint axis-angle representations for rotations.

**Comment by SreehariRajan** [comment]
"Through optimization," , you mean IK ?

**Comment by RobinWitch** [comment]
Yes, it uses Vposer as IKEngine. You can dircect run this command `python libs/human_body_prior/tutorials/mdm_motion2smpl.py --input your_npy_file_path --output your_file_name.npz`
